www/docs/reactjs/useInfiniteQuery.md

```diff
@@ -90,7 +89,7 @@ import { trpc } from '../utils/trpc';
 export function MyComponent() {
   const utils = trpc.useContext();
 
-  const myMutation = trpc.useMutation('infinitePosts.add', {
+  const myMutation = trpc.proxy.infinitePosts.add.useMutation({
     onMutate({ post }) {
       await utils.cancelQuery(['infinitePosts']);
```

Comment:
unsure if `utils` requires any changes like `utils.proxy.cancelQuery()`

Reply:
That's the plan but it hasn't been implemented yet (see https://github.com/trpc/trpc/issues/2113) so it's ok to leave this as is